### PR TITLE
Chore: Update translation readme file

### PR DESCRIPTION
**Pull request by matin-deriv**
Update translation readme file

### CODE REVIEW

File: scripts/README.md [hunk 1]
```diff
@@ -13,8 +13,8 @@
 - Install `Homebrew` by `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
 - Install and follow the instructions here: https://support.crowdin.com/cli-tool/#installation
 - **Important:** Please use version 3 of the crowdin cli-tool
-- Get [Crowdin API Key](https://crowdin.com/project/binary-static/settings#api) and add this line to your _.bash_profile_: `export CROWDIN_API_KEY='put API Key here'`<br/>
-  **IMPORTANT!** This key gives full access to all your Crowdin project data. Treat this just like a password and never push it to any public repo.
+- Get [Crowdin API Key](https://crowdin.com/settings#api-key) and add this line to your _.bash_profile_ / _.zshrc_ using terminal: `export CROWDIN_API_KEY='put API Key here'`<br/>
+  **Improtant!** This key gives full access to all your Crowdin project data. Treat this just like a password and never push it to any public repo.
```
Comment (yashim-deriv) [inline scripts/README.md]:
```suggestion
  **Important!** This key gives full access to all your Crowdin project data. Treat this just like a password and never push it to any public repo.
```

File: scripts/README.md [hunk 2]
```diff
@@ -13,8 +13,8 @@
 - Install `Homebrew` by `ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
 - Install and follow the instructions here: https://support.crowdin.com/cli-tool/#installation
 - **Important:** Please use version 3 of the crowdin cli-tool
-- Get [Crowdin API Key](https://crowdin.com/project/binary-static/settings#api) and add this line to your _.bash_profile_: `export CROWDIN_API_KEY='put API Key here'`<br/>
-  **IMPORTANT!** This key gives full access to all your Crowdin project data. Treat this just like a password and never push it to any public repo.
+- Get [Crowdin API Key](https://crowdin.com/settings#api-key) and add this line to your _.bash_profile_ / _.zshrc_ using terminal: `export CROWDIN_API_KEY='put API Key here'`<br/>
```
Comment (yashim-deriv) [inline scripts/README.md]: Maybe add a note on using the API v2 token

**Review by yashim-deriv — COMMENTED** [review]
The rest LGTM

File: scripts/README.md (hunk 1 above)
Comment (matin-deriv) [reply scripts/README.md]: 🤦🏼‍♂️

File: scripts/README.md (hunk 2 above)
Comment (matin-deriv) [reply scripts/README.md]: Yeah good point 👌🏼

**Review by yashim-deriv — APPROVED** [review]
LGTM

**Review by mahdiyeh-deriv — APPROVED** [review]

**Pull request merged**